test/bdd/features/jwt.feature

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@verifiable_jwt
+Feature: Issue and verify Verifiable Credential and Verifiable Presentation with JWS proof
+  @issue_vc_vp_jwt_ed25519
+  Scenario: Issue VC - verify VC - create VP - verify VP with Ed25519 JWS proof
+    When "Berkley" issues VC at "2022-04-12" regarding "Master Degree" to "Alice" with "Ed25519" JWS proof
+    Then "Alice" receives the VC and verifies it
+    Then "Alice" embeds the VC into VP with "Ed25519" JWS proof
+    Then "Alice" verifies VP
+
+  @issue_vc_vp_jwt_ecdsaSecp256r1
```

Comment:
No need to duplicate steps you can use something like this https://github.com/hyperledger/aries-framework-go/blob/main/test/bdd/features/didexchange_public_did.feature#L45

Reply:
great catch! udpated